### ISSUE TITLE
Chef v13 Breaks cop_base::packages

### ISSUE THREAD

**Issue by jerrywardlow** [copious-cookbooks/base#15]
Chef v13.0.113 breaks packages recipe. Last stable v12 release v12.19.36 operates correctly.

```
Recipe: cop_base::packages
         * yum_package[autoconf] action install

           ================
           Error executing action `install` on resource 'yum_package[autoconf]'
           ================

           NoMethodError
           -------------
           undefined method `split' for ["--enablerepo=epel"]:Array

           Resource Declaration:
           ---------------------
           # In /tmp/kitchen/cache/cookbooks/cop_base/recipes/packages.rb

             3:        package pkg do
             4:             action  :install
             5:             options "--enablerepo=#{node['epel']['repo_name']}" if node['platform_family'] == 'rhel'
             6:         end
             7:     end

           Compiled Resource:
           ------------------
           # Declared in /tmp/kitchen/cache/cookbooks/cop_base/recipes/packages.rb:3:in `block in from_file'

           yum_package("autoconf") do
             package_name "autoconf"
             action [:install]
             default_guard_interpreter :default
             declared_type :package
             cookbook_name "cop_base"
             recipe_name "packages"
             options ["--enablerepo=epel"]
           end

           System Info:
           ------------
           chef_version=13.0.113
           platform=centos
           platform_version=7.3.1611
           ruby=ruby 2.4.1p111 (2017-03-22 revision 58053) [x86_64-linux]
           program_name=chef-client worker: ppid=26495;start=19:27:30;
           executable=/opt/chef/bin/chef-client
```

**Comment by jerrywardlow** [comment]
This issue has been fixed by Chef v13.0.118. The current Chef v12 pin remains until all cookbooks can be evaluated against Chef v13.

**Comment by jerrywardlow** [comment]
Closed as all cookbooks have been verified with Chef v13
